### PR TITLE
Implemented next.skip(int)

### PR DESCRIPTION
**Pull request by hallas**
@wejendorp @Nokk @thorning 

Added the next.skip(int) function in order to skip 1 or more steps in a flow. Please review this code, test it out and make sure it works like you expect.

### CODE REVIEW

**Comment by wejendorp** [comment]
This seems too much like a goto, and using different paths for different conditions sounds like something where each path should be wrapped into its own function instead.

**Comment by hallas** [comment]
@wejendorp there's a line right?

If it's just ONE step that you want to skip, and it doesn't make sense on it's own, then I think this is a useful little thing.

File: index.js [hunk 1]
```diff
@@ -155,6 +155,13 @@ exports.step = function(funcs, onerror) {
 		};
 	};
 
+  next.skip = function (step) {
+    pointer += step;
+    return function (err, value) {
+      next(err, value);
+    }
```
Comment (mafintosh) [inline index.js:162]: Any reason we can't just `return next` here?

**Comment by mafintosh** [comment]
@hallas Some how this didn't show up in my Gmail. I just reviewed the changes. Besides some the comment it's looking GTM.

File: index.js (hunk 1 above)
Comment (hallas) [inline index.js:162]: I think we can just return next yes.

**Comment by hallas** [comment]
No prob :-)

**Pull request merged**